configure

```diff
@@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
 # Invoke python_config and set up symlinks to python includes
 (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
 
-## Set up Cuda-related environment settings
-
-while [ "$TF_NEED_CUDA" == "" ]; do
```

Comment:
One thing I noticed is that after running `CUDA_VERSION=7.5 bazel fetch --config=cuda //...` (which I put in the `./configure` script), the `local_config_cuda` external repository does not show up in `bazel-tensorflow/external` until after building certain targets.

Further, I found that I need to add `CUDA_VERSION=7.5` to the `bazel build` invocation in order for the `@local_config_cuda` to be configured with the `CUDA_VERSION=7.5`, which seems to suggest that `@local_config_cuda` wasn't fetched by `bazel fetch //...`. Is this a bug or is the `bazel fetch` invocation incorrect here?